src/main/webapp/i18n/de/result.json

```diff
@@ -32,7 +32,7 @@
             "rated": "Bewertet",
             "hasFeedback": "Enthält Feedback",
             "assessmentType": "Bewertungstyp",
-            "assessor": "Reviewer",
+            "assessor": "Gutachter:in",
```

Comment:
s.o. consistent translation :)

Reply:
for the time being changed to `Reviewer:in` to stay consistent. Might be changed in any place afterwards.